src/main/webapp/help.html

```diff
@@ -4,6 +4,8 @@
   
   The text you enter in the text box will be executed as a PowerShell file, and a build
   will be considered a failure if at the end of the execution $LastExitCode is not 0.
+
+  By default PowerShell uses profile(s) which can be disabled to avoid any potential conflict and provide a clean shell.
```

Comment:
"By default, PowerShell processes profiles scripts at startup.  This can be disabled to improve startup time, avoid any potential conflict and provide a clean shell."